fixed react player alt image and fixed padding for caption

vishnus17 opened the pull request:
**Notes for Reviewers**

This PR fixes #75 


**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 
<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

Yashsharma1911 commented:
nice work @vishnus17 as you are there can you do a small change, you can see in this below image corner of the player is not in a proper curve can you fix this too

![Screenshot 2023-02-25 at 3 24 27 AM](https://user-images.githubusercontent.com/71271069/221300403-a061850a-e772-4ddb-b025-b6b7e578defb.png)

vishnus17 commented:
I don't think I can do that. The image fixing I did was with canva. It was just adding a rectangle over the Login button which I noticed when I was signing up for the early access. I'm not really that much of a graphics guy 😅

Avi-88 reviewed (changes requested):
Nice work @vishnus17. About what @Yashsharma1911 suggested , you dont need to edit the image itself @vishnus17. Just make it so that the border radius for that component is not in % but a static value like for e.g 10px. You can inspect using dev tools and find the class which applies the said styling.

Nikhil-Ladha commented on site/src/App.style.js:
```diff
@@ -218,7 +218,7 @@ export const Main = styled.main`
   .caption {
     color: ${({ theme }) => theme.text};
-    margin-top: 0.75rem;
+    margin-top: 3rem;
```
This is logically not needed, this is a caption and should be close to the image and not same as the callout text below it.

Nikhil-Ladha commented:
> nice work @vishnus17 as you are there can you do a small change, as you can see the corner of the player in the below image is not in a proper curve can you fix this too
> 
> ![Screenshot 2023-02-25 at 3 24 27 AM](https://user-images.githubusercontent.com/71271069/221300403-a061850a-e772-4ddb-b025-b6b7e578defb.png)

Where do you see that the corner doesn't have the same radius as the image? 🤔 
Seems same to me, or am I _just_ not able to see it

Yashsharma1911 commented:
> > nice work @vishnus17 as you are there can you do a small change, as you can see the corner of the player in the below image is not in a proper curve can you fix this too
> > ![Screenshot 2023-02-25 at 3 24 27 AM](https://user-images.githubusercontent.com/71271069/221300403-a061850a-e772-4ddb-b025-b6b7e578defb.png)
> 
> Where do you see that the corner doesn't have the same radius as the image? 🤔 Seems same to me, or am I _just_ not able to see it

yes it is actually not properly curved this is because radius is given in % 

### current
![Screenshot 2023-02-28 at 3 02 12 PM](https://user-images.githubusercontent.com/71271069/221872035-01c85e97-c6a2-441d-aff7-c434087110c2.png)

### Desired 
![Screenshot 2023-02-28 at 3 03 15 PM](https://user-images.githubusercontent.com/71271069/221872105-0ee16ac6-81c6-49ac-bed7-1f67006bb792.png)

leecalcote commented:
@vishnus17 following up a final time here... is this something that you intend to complete?

adithyaakrishna commented:
@vishnus17 The fix should be to change the value from 2.5% to 15px, that should fix the case of two backgrounds being visible at the same time as well :)

vishnus17 commented:
Hi guys. Sorry I was having really busy weeks lately.
@leecalcote . Yes I'll complete this.
@adithyaakrishna Thanks for the help.
I'll complete this by this week.

leecalcote commented:
Sounds great.

vishnus17 commented:
Hi @Yashsharma1911, @Nikhil-Ladha , @leecalcote , @adithyaakrishna . 
I've removed the padding for the caption element and adjusted the border-radius of the react player image.
Please have a final look and see if its good to be merged.
Thanks.

adithyaakrishna reviewed (approved):
LGTM

adithyaakrishna commented:
@Nikhil-Ladha Could you please merge this PR?

Nikhil-Ladha commented on site/src/App.style.js:
```diff
@@ -197,7 +197,7 @@ export const Main = styled.main`
 
     .react-player__preview {
-      border-radius: 2.5%;
+      border-radius: 1.5%;
```
```suggestion
      border-radius: '2rem';
```

Nikhil-Ladha commented on site/src/App.style.js:
```diff
@@ -197,7 +197,7 @@ export const Main = styled.main`
 
     .react-player__preview {
-      border-radius: 2.5%;
+      border-radius: '2rem';
```
```suggestion
      border-radius: 1.5%;
```
